HIRING.md

```diff
@@ -48,3 +48,26 @@ We’re obviously trying to assess the candidate but remember they are assessing
 
 ## Assignment
 The candidate will be asked to make an assigment with a 24 hour deadline. He'll get to pick the best day to receive the details of the assignment then the clock will start ticking. The assignment will consist in making a simple app to display a list of items that comes from an API. The list should be able to be accessed offline.
+
+# Job description for an iOS position
+
+### A bit about you
+- 01 Experience with building top level applications for Apple iOS/OS X, and started peeping into Watch OS and tvOS.
```

Comment:
Let's keep it simple.
`01 Experience with building top level applications for iOS.`